library/network_connections.py

```diff
@@ -818,12 +818,14 @@ def connection_compare(
             try:
                 con_a.normalize()
             except Exception:
+                # It's already checked normalize is successful.
```

Comment:
@thom311 what kind of exceptions are ignored here?

Reply:
Do you mean, which exceptions `normalize()` is expected to throw? Probably they are all subtypes of `GLib.Error`. But I don't know, and I don't want to find out -- because even if I check *now*, it's not clear that this check will hold for the future. The code *really* wants to catch and ignore *any* errors (that is, all `Exception` types -- but not all `BaseException`).

The code seems pretty clear in that. This is not a bug but done unintentionally. The code wants to normalize the profile, and if that fails, ignore the error. *Any* error.

The new code comment seems not clear to me. The proper comment would be `# We ignore any errors that might happen`, which seems a pretty meaningless thing to say. This linter rule seems questionable to me, but OK, it needs to be worked around.

---

In the commit message, I would not say "fix issues" (but false positives or warnings) -- or does the patch also fix actual bugs? In that case, the actual bug fixes should be separate commits.

Reply:
I mean what can go wrong with normalization that it can be ignored but the connection is still fine without normalization so that we can safely ignore the error.

Reply:
> In the commit message, I would not say "fix issues" (but false positives or warnings) -- or does the patch also fix actual bugs? In that case, the actual bug fixes should be separate commits.

Well, maybe "issues" means something quite benign (like a warning). In my ear, it sounds severe to fix "issues". I would not use the wording "fix issue" for adding a code comment. If the patch fixes actual issues, this patch should not be cluttered with low-severity changes. If it doesn't fix any bugs, the word "issue" seems unnecessarily alarming.